packages/store/src/lib/operators.ts

```diff
@@ -51,3 +53,24 @@ export function filterNil<T>(): OperatorFunction<T, NonNullable<T>> {
     (value: T): value is NonNullable<T> => value !== null && value !== undefined
   );
 }
+
+// TODO infer the IDType
+type IDType = string | number;
+
+export function ofTypes<T extends EntityAction<IDType>>(
+  actionOrActions?: EntityActions | EntityActions[]
+): OperatorFunction<T, IDType[] | EntityAction<IDType>> {
+  if (isUndefined(actionOrActions)) {
+    return map((action) => action);
+  }
+
+  const transform = Array.isArray(actionOrActions)
```

Comment:
I'd always return the whole action object. In the future it main contain additional properties